pkg/app/instances/gce.go

```diff
@@ -61,6 +61,26 @@ func NewGCEInstanceManager(cfg Config, service *compute.Service, nameGenerator N
 	}
 }
 
+func (m *GCEInstanceManager) ListZones() (*apiv1.ListZonesResponse, error) {
+	res, err := m.Service.Zones.List(m.Config.GCP.ProjectID).Context(context.TODO()).Do()
+
+	if err != nil {
+		return nil, toAppError(err)
+	}
+
+	var items []*apiv1.Zone
+	for _, i := range res.Items {
```

Comment:
It might be good to change to `for _, i := ...` to `for _, item :=` to prevent confuse with i for index.

Reply:
done